org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java

```diff
@@ -35,14 +36,14 @@ public void testGetIdClass() {
   @Test
   public void testGetTypeNameByClass() {
     assertEquals("scout.FixtureUuId", m_inventory.getTypeName(FixtureUuId.class));
-    assertNull(m_inventory.getTypeName(FixtureLongId.class));
+    assertNull(m_inventory.getTypeName(FixtureLongIdWithoutTypeName.class));
     assertNull(m_inventory.getTypeName((Class<? extends IId>) null));
   }
 
   @Test
   public void testGetTypeNameByInstance() {
     assertEquals("scout.FixtureUuId", m_inventory.getTypeName(FixtureUuId.of("3c5a66be-12b4-45c6-9e59-8c31cf92dcfb")));
-    assertNull(m_inventory.getTypeName(FixtureLongId.of(100L)));
+    assertNull(m_inventory.getTypeName(FixtureLongIdWithoutTypeName.of(100L)));
```

Comment:
you could add another testcase for FixtureLongId

Reply:
done